src/test/javascript/spec/component/lecture/lecture-attachments.component.spec.ts

```diff
@@ -379,4 +380,23 @@ describe('LectureAttachmentsComponent', () => {
         expect(comp.attachmentToBeCreated.link).toBe(myBlob1.name);
         expect(attachmentServiceFindAllByLectureIdStub).toHaveBeenCalledOnce();
     }));
+
+    describe('isViewButtonAvailable', () => {
+        it('should return true if the attachment link ends with .pdf', () => {
+            const attachment = { id: 1, link: 'example.pdf', attachmentType: 'FILE' } as Attachment;
+            expect(comp.isViewButtonAvailable(attachment.link!)).toBeTrue();
+        });
+
+        it('should return false if the attachment link does not end with .pdf', () => {
+            const attachment = { id: 2, link: 'example.txt', attachmentType: 'FILE' } as Attachment;
+            expect(comp.isViewButtonAvailable(attachment.link!)).toBeFalse();
+        });
+
+        it('should return false for other common file extensions', () => {
```

Comment:
Please user a proper parameterised test. (Search for `it.each` in our code base for a few examples)

Reply:
Done